src/game/server/swarm/asw_marine.cpp

```diff
@@ -1419,7 +1419,12 @@ int CASW_Marine::OnTakeDamage_Alive( const CTakeDamageInfo &info )
 				Msg( "  but all ignored, since it's from a team mate\n" );
 			return 0;
 		}
-		else
+		//make hornet and smart bomb no longer friendlyfire	
+		else if (!ASWDeathmatchMode() && pWeapon && (pWeapon->Classify() == CLASS_ASW_HORNET_BARRAGE || pWeapon->Classify() == CLASS_ASW_SMART_BOMB))
+			{
+    			return 0;
+			}			
+else
```

Comment:
```suggestion
		else
```

it should be over there with its friends